app/helpers/time_options_helper.rb

```diff
@@ -20,6 +20,10 @@ def format_full_date_and_time(datetime)
     datetime.strftime('%-l:%M%P on %-d %B %Y')
   end
 
+  def format_full_date(datetime)
```

Comment:
Could you add some tests for this method? (There are equivalents for format_full_date_and_time)

Reply:
Sure, have added now.